Include/httpClient/httpClient.h

```diff
@@ -223,6 +223,18 @@ STDAPI HCHttpCallPerformAsync(
     _Inout_ AsyncBlock* asyncBlock
     ) HC_NOEXCEPT;
 
+/// <summary>
+/// Get hc_call_handle_t from an HCHttpCallPerformAsync call.
+/// Use HCHttpCallResponse* to get information from the hc_call_handle_t.
+/// </summary>
```

Comment:
If you are cleaning up the hc_call_handle_t object on failure, make that clear in the comments.